Review "Task" page (#547)
grafana/writers-toolkit: docs/sources/structure/topic-types/task/index.md

  You can link to the longer concept topic from this introduction.

- **Before you begin: (optional)** Add links to tasks that need to be completed before the current one.
  The links might sometimes be unrelated to the product, such as “Have this thing at hand”.

  Additionally, this section can include decisions the user should make or permissions they need to confirm before starting the task.

  If there are no prerequisites, don't include this section.

- **Steps heading**: The steps heading indicates that the steps are about to begin.

- **Stem sentence:** The stem sentence introduces the steps.

  Use the following convention when you write a stem sentence: To [name of task], follow these steps:
  For example: To create a dashboard, follow these steps:

  For more information about stem sentences, refer to [Procedures](https://developers.google.com/style/procedures) in the [Google developer documentation style guide](https://developers.google.com/style).

- **Steps:** Users are provided with a directive through numbered steps.

  Write steps so that they contain one action, or possibly two related actions, such as _Copy and paste a value_ or _Save and quit the program._

>   Unless a sentence instructs the reader to act, it isn't a step.

Reviewer: For me... this is a weird sounding sentence.

Could it be rewritten to say something like:

`A step must instruct the reader to act or perform a specific task.`

or something similar?

Author: ```suggestion
  A step must instruct the reader to perform a specific task.
```
